tests/integration/eZ/API/RichTextFieldTypeIntegrationTest.php

```diff
@@ -624,6 +634,127 @@ public function testConvertRemoteObjectIdToObjectId($test, $expected)
+        $content = $contentService->createContent(
+            $testContentCreateStruct,
+            [$locationService->newLocationCreateStruct(2)]
+        );
+        $content = $contentService->publishVersion(
+            $content->versionInfo
+        );
+        $urlIds = $this->getUrlIdsForContentObjectAttributeIdAndVersionNo(
+            $content->getField('description')->id,
+            $content->contentInfo->currentVersionNo
+        );
+
+        $xmlDocument = $this->createXmlDocumentWithExternalLink('https://support.ez.no/');
+        $contentUpdateStruct = $contentService->newContentUpdateStruct();
+        $contentUpdateStruct->setField('description', $xmlDocument, 'eng-GB');
+        $contentDraft = $contentService->updateContent(
+            $contentService->createContentDraft($content->contentInfo)->versionInfo,
+            $contentUpdateStruct
+        );
+        $content = $contentService->publishVersion($contentDraft->versionInfo);
+        $urlIdsAfterUpdate = $this->getUrlIdsForContentObjectAttributeIdAndVersionNo(
+            $content->getField('description')->id,
+            $content->contentInfo->currentVersionNo
+        );
+
+        $this->assertNotContains(reset($urlIds), $urlIdsAfterUpdate);
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\ContentType\ContentType
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    private function createContentTypeForTestExternalLinkStoringAfterUpdate(): ContentType
+    {
+        $repository = $this->getRepository();
+        $contentTypeService = $repository->getContentTypeService();
+        $contentTypeCreateStruct = $contentTypeService->newContentTypeCreateStruct('test-RichText');
+        $contentTypeCreateStruct->mainLanguageCode = 'eng-GB';
+        $contentTypeCreateStruct->names = ['eng-GB' => 'Test'];
+
+        $fieldDefinitionCreateStruct = $contentTypeService->newFieldDefinitionCreateStruct(
+            'description',
+            'ezrichtext'
+        );
+        $fieldDefinitionCreateStruct->names = ['eng-GB' => 'Title'];
+        $fieldDefinitionCreateStruct->fieldGroup = 'main';
+        $fieldDefinitionCreateStruct->position = 1;
+        $fieldDefinitionCreateStruct->isTranslatable = true;
+        $contentTypeCreateStruct->addFieldDefinition($fieldDefinitionCreateStruct);
+
+        $contentGroup = $contentTypeService->loadContentTypeGroupByIdentifier('Content');
+        $contentTypeDraft = $contentTypeService->createContentType(
+            $contentTypeCreateStruct,
+            [$contentGroup]
+        );
+
+        $contentTypeService->publishContentTypeDraft($contentTypeDraft);
+
+        return $contentTypeService->loadContentType($contentTypeDraft->id);
+    }
+
+    /**
+     * @param $contentObjectAttributeId
+     * @param $versionNo
```

Comment:
those attributes provide no value, you can drop them

Reply:
> those attributes provide no value, you can drop them

Still unresolved. My point was:
```suggestion
```
the PHPDoc was not even correct anyway

Reply:
Deleted, I was checking another file